framework/src/wifi_manager/wifi_manager.c

```diff
@@ -507,37 +447,18 @@ static const wifimgr_handler g_handler[] = {
 
 
 #ifndef CONFIG_WIFIMGR_DISABLE_DHCPS
-/* wifi manager stores only one mac address of a node while it is running on softap mode
- * because most of wi-fi chipset that ported to TizenRT supports to join a device.
- * these dhcp function might be modified later when
- * a chipset that supports multiple connections are ported
- */
-#define WIFIMGR_IP4_ZERO 0
-#define WIFIMGR_MAC_ZERO {0, 0, 0, 0, 0, 0}
-// g_dhcp_list is protected by g_manager_info.disconn_lock
-// because it's called while wifimanager is softap mode
-static dhcp_node_s g_dhcp_list = {WIFIMGR_IP4_ZERO, WIFIMGR_MAC_ZERO};
-
-int _dhcps_add_node(dhcp_node_s *node)
+static void _wifi_dhcps_event(dhcp_evt_type_e type, void *data)
 {
-	if (memcmp(g_dhcp_list.macaddr, node->macaddr, 6) == 0) {
-		g_dhcp_list.ipaddr = node->ipaddr;
-		return DHCP_EXIST;
-	}
-	g_dhcp_list.ipaddr = node->ipaddr;
-	memcpy(g_dhcp_list.macaddr, node->macaddr, 6);
-	return DHCP_OK;
-}
-
-void _dhcps_remove_list(void)
-{
-	g_dhcp_list.ipaddr = 0;
-	for (int i = 0; i < 6; i++) {
-		g_dhcp_list.macaddr[i] = 0;
+	WM_LOG_START;
+	if (type == DHCP_ACK_EVT) {
+		_wifimgr_msg_s msg = {EVT_DHCPS_ASSIGN_IP, data};
+		WIFIMGR_CHECK_RESULT_NORET(_handle_request(&msg), "[WM] handle dhcpd event fail\n");
 	}
+	return;
```

Comment:
it's void.

Reply:
It just returns, because it is void. Please let me know your suggestion details. Thanks!!

Reply:
It is end of code, return is useless here.

Reply:
If not wrong, I think it depends on the taste of developer. In this way, we can look at the return statement and see that it doesn't return anything at a glance.